app/helpers/concerns/foreman_leapp/remote_execution_helper_extension.rb

```diff
@@ -3,13 +3,18 @@
 module ForemanLeapp
   module RemoteExecutionHelperExtension
     def job_invocation_task_buttons(task)
-      return super unless @job_invocation.remote_execution_feature&.label == 'leapp_remediation_plan'
+      job_invocation = task.task_groups.find { |group| group.class == JobInvocationTaskGroup }.job_invocation
```

Comment:
Wouldn't just `@job_invocation.reload` work?

Reply:
No, it would work without problems, rubocop does not like using instance variables in helpers:


```
RuboCop failed!
/home/runner/work/foreman_leapp/foreman_leapp/app/helpers/concerns/foreman_leapp/remote_execution_helper_extension.rb:6:27: C: Rails/HelperInstanceVariable: Do not use instance variables in helpers.
      return super unless @job_invocation.remote_execution_feature&.label == 'leapp_remediation_plan'
                          ^^^^^^^^^^^^^^^
```